forc-plugins/forc-client/src/ops/pkg_util.rs

```diff
@@ -0,0 +1,54 @@
+use std::{collections::HashMap, path::Path};
+
+use anyhow::Result;
+use forc_pkg::{self as pkg, manifest::ManifestFile, BuildOpts, BuildPlan};
+use pkg::{manifest::MemberManifestFiles, BuiltPackage, PackageManifestFile};
+
+fn built_pkgs(
+    build_opts: BuildOpts,
+    member_manifests: &MemberManifestFiles,
+) -> Result<HashMap<String, BuiltPackage>> {
+    let built = forc_pkg::build_with_options(build_opts)?;
```

Comment:
Ahh, I'm just realizing that we end up building *all* members here (not just contracts). Ideally, we would do the build with only *contract* members as outputs, not all members as outputs. This is probably fine for now, but we should open an issue to address this in a future PR. We might want to expose an alternative `forc_pkg::build_contracts_with_opts` or similar, or allow for specifying the set of desired outputs.